### PR TITLE
update to the latest plugins

### PR DESCRIPTION
**Pull request by edewit**
fixing issue [RHMAP-6292](https://issues.jboss.org/browse/RHMAP-6292)
depended on https://github.com/feedhenry/fh-js-sdk/pull/162

### CODE REVIEW

File: www/config.json [hunk 1]
```diff
@@ -1,53 +1,54 @@
...
-    "url": "https://github.com/apache/cordova-plugin-camera.git#r0.3.2"
+    "id": "cordova-plugin-camera",
+    "version": "2.2.0",
+    "url": "https://github.com/apache/cordova-plugin-camera.git#rel/2.2.0"
   },
   {
-    "id": "com.phonegap.plugins.barcodescanner",
-    "version": "2.0.0"
+    "id": "phonegap-plugin-barcodescanner",
+    "version": "2.0.0",
```
Comment (corinnekrych) [inline www/config.json]: this should be 4.1.0

**Comment by nialldonnellyfh** [comment]
@edewit Can you bump the package.json version

**Comment by corinnekrych** [comment]
+1
you need  rebase and then ready to merge

**Comment by edewit** [comment]
@corinnekrych @nialldonnellyfh rebased, good to merge?

**Comment by nialldonnellyfh** [comment]
@edewit Needs a package.json version bump.

Also need to update https://github.com/feedhenry/appforms-project-client which is the minified version of this app.

**Comment by edewit** [comment]
@nialldonnellyfh the version number was already increased when I rebased can't this change be part of the same release? How do I create a minified version of this project?

**Comment by nialldonnellyfh** [comment]
@edewit It should be a different version of the client app but it will be in the same release (3.11.0) when added to fh-template-apps (1.2.0) 

I am missing a changelog entry for 1.1.0 , I will add this.

See https://github.com/feedhenry/Appforms-Template-v3#overview for generating the minified version.

**Comment by edewit** [comment]
@nialldonnellyfh like that?

**Comment by nialldonnellyfh** [comment]
@edewit Yes, my missing PR is https://github.com/feedhenry/Appforms-Template-v3/pull/16

**Pull request merged**